node/build-css.js

```diff
@@ -55,7 +57,12 @@ const buildCSS = async (args) => {
 
     const cssFilesToRead = importFrom;
 
-    const postcssPlugins = [postcssImport];
+    const postcssPlugins = [
+      postcssImport,
+      postcssCustomMedia({
```

Comment:
Er det kun docs der benytter dette? Vil man ikke miste custom queries i eb-repoet, hvis de konverteres her?

Reply:
Jo, de vil ikke stå i den færdige _eb-designsystem.css_ og det er sådan set pointen :) 
Det er ikke en css standard, så hvis man vil konsumere designsystemet kræver det pt at man kører postcss over det - det er ikke så aftager-venligt.
Det vil, for ekstrabladet, betyde at den færdigbyggede fil vil være minimalt større, da nogle ting ikke vil blive slettet i bygget af de device specifikke filer. (som en sidenote overvejer jeg også om vi overhovedet "længere" behøver 3 css filer)

Reply:
Alright, jeg så bare de bliver benyttet i en vis udstrækning på eb-repo. Men det er nok primært fra frontend/css-main/main/css/variables/_mediaqueries.css .